index.html

```diff
@@ -9,8 +9,32 @@
 		<link rel="stylesheet" href="styles.css">
 	</head>
 
-	<body>
+	<body class="body">
+		<header>
+			<div class="header">
+					<img src="image_geometry_1.svg" alt="geometric image" class="image_header">
+					<img src="ic_menu.svg" alt="menu" class="ic_image">
+			</div>
+		</header>
 
+		<div class="headline">
+			<div class="name">
+				Akumu
+				Bavon
+			</div>
```

Comment:
Hi @Bavon101 ✋. Please use `h1` tag instead of `div`. You can check [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) for more information.